lib/app/utils/custom_error_screen.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:get/get.dart';
+import 'package:ultimate_alarm_clock/app/modules/settings/controllers/theme_controller.dart';
+import 'package:ultimate_alarm_clock/app/utils/constants.dart';
+
+class CustomErrorScreen extends StatefulWidget {
+  const CustomErrorScreen({
+    super.key,
+    required this.errorDetails,
+  });
+
+  final FlutterErrorDetails? errorDetails;
+
+  @override
+  State<CustomErrorScreen> createState() => _CustomErrorScreenState();
+}
+
+class _CustomErrorScreenState extends State<CustomErrorScreen> {
+  ThemeController themeController = Get.put<ThemeController>(ThemeController());
+
+  @override
+  Widget build(BuildContext context) {
+    var height = Get.height;
+    var width = Get.width;
+    return Scaffold(
+      backgroundColor: themeController.isLightMode.value
+          ? kLightPrimaryBackgroundColor
+          : kprimaryBackgroundColor,
+      body: SafeArea(
+        child: SingleChildScrollView(
+          child: Padding(
+            padding: EdgeInsets.fromLTRB(16, height * 0.2, 16, 16),
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                SvgPicture.asset(
```

Comment:
A title/heading signifying an error would be great!

Reply:
Okay, do you mean an "Error Occured" title at the top?

Reply:
Yeah